Sources/Shared/Requests/Request+ConnectedApp.swift

```diff
@@ -23,7 +23,7 @@
 //  THE SOFTWARE.
 //
 
-public extension Request {
+public extension Request where ModelType: ConnectedApp {
```

Comment:
@jasonhawkins In this case model for `connectedApps()` method is wrong and always empty. There should be array of `ConnectedApp`

I used this code
```
let request = Request<[ConnectedApp]>.connectedApps()
....
case .success(let response):
   response.model // Array of ConnectedApp
```
and was all fine.

Reply:
Also for `deleteConnectedApp` seems should be `VIMNullResponse` model instead of `ConnectedApp`, because there no info in response

`Request<VIMNullResponse>.deleteConnectedApp(.facebook)`

Reply:
Hey Anton, you're totally right. Sorry about that. I've just pushed changes that will constrain the request objects to the correct types, and added some tests to validate those types.

You should now be able to use those request objects without having to specify `<[ConnectedApp]>` or `<VIMNullResponse>`, etc.